austrakka/utils/options.py

```diff
@@ -91,12 +91,12 @@ def opt_sample_id(**attrs: t.Any):
     defaults = {
         'required': True,
         'help': 'name',
+        'multiple': True,
```

Comment:
Why move this from overridable defaults?

Reply:
Sorry I completely misread this, you moved it to overridable defaults. Looks good